inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/fused_conv_eltwise_gpu_imad.cl

```diff
@@ -257,9 +410,33 @@ KERNEL (fused_convolution_eltwise_gpu_imad)(
 #undef IN_BLOCK_WIDTH
 #undef IN_BLOCK_HEIGHT
 #undef PACK
+#undef TYPE_N_
+#undef TYPE_N
 #undef AS_TYPE_N_
 #undef AS_TYPE_N
+#undef INPUT0_TYPE_4
 #undef AS_INPUT0_TYPE_4
+
+#ifdef NON_ZERO_INPUT0_PAD_BEFORE
```

Comment:
I think you can avoid this checks and in all cases can use #undef instead of #ifdef - #undef - #endif.